app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -108,8 +108,7 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   @Inject
   lateinit var inputFields: InputFields
 
-  @Inject
-  lateinit var editPatientConfig: EditPatientConfig
+  private val editPatientConfig = EditPatientConfig()
```

Comment:
We should not be constructing this directly and they should be injected. Given that this is only a temporary thing, it's fine for now.